src/main/java/com/google/devtools/build/lib/starlarkbuildapi/test/CoverageConfigurationApi.java

```diff
@@ -0,0 +1,34 @@
+package com.google.devtools.build.lib.starlarkbuildapi.test;
+
+import com.google.devtools.build.docgen.annot.DocCategory;
+import com.google.devtools.build.lib.cmdline.Label;
+import javax.annotation.Nullable;
+import net.starlark.java.annot.StarlarkBuiltin;
+import net.starlark.java.annot.StarlarkMethod;
+import net.starlark.java.eval.StarlarkValue;
+
+@StarlarkBuiltin(
+    name = "coverage",
+    category = DocCategory.CONFIGURATION_FRAGMENT,
+    doc = "A configuration fragment representing the coverage configuration.")
+public interface CoverageConfigurationApi extends StarlarkValue {
+
+  @StarlarkMethod(
+      name = "output_generator",
+      allowReturnNones = true,
+      structField = true,
+      doc =
+          "Returns label pointed to by "
```

Comment:
Can you add a note saying that this will be None if coverage collection isn't enabled?

Reply:
Done, I also slightly reworded the comment.